monkey/infection_monkey/command_builders/windows_agent_command_builder.py

```diff
@@ -68,15 +68,17 @@ def build_run_command(self, run_options: WindowsRunOptions):
             if run_options.shell == WindowsShell.CMD:
                 self._command += "cmd.exe /c "
 
-        if run_options.shell == WindowsShell.POWERSHELL:
-            set_otp = self._set_otp_powershell
-        elif run_options.shell == WindowsShell.CMD:
-            set_otp = self._set_otp_cmd
-
-        self._command += f"{set_otp()} {str(run_options.agent_destination_path)} "
+        if run_options.include_otp:
+            if run_options.shell == WindowsShell.POWERSHELL:
+                set_otp = self._set_otp_powershell
+            elif run_options.shell == WindowsShell.CMD:
+                set_otp = self._set_otp_cmd
+            self._command += f"{set_otp()} {str(run_options.agent_destination_path)}"
```

Comment:
```suggestion
            self._command += f"{set_otp()} "
    self._command += f"{str(run_options.agent_destination_path)} "
```

I think this you go like this but check.